### PR TITLE
Update search with creds and change worker views

### PR DESCRIPTION
**Pull request by PhilSharland**

### CODE REVIEW

File: routes/home.rb [hunk 1]
```diff
@@ -86,6 +85,31 @@ def h(text)
   erb :search, locals: { title: 'Search' }
 end
 
+# Download
+get '/download' do
+  authenticate!
+  role = session[:role]
+
+  if role == 'manager'
```
Comment (johntopleyons) [inline routes/home.rb]: Use `.eql?` for string comparisons.

File: routes/home.rb [hunk 2]
```diff
@@ -86,6 +85,31 @@ def h(text)
...
+
+    RestClient::Request.execute(method: :get,
+                                user: SPRING_SECURITY_USER_NAME,
+                                password: SPRING_SECURITY_USER_PASSWORD,
+                                url: 'http://' + CENSUS_FSDR_HOST + ':' + CENSUS_FSDR_PORT + '/fieldforce/allEmployeeCsv') do |download_file, _request, _result, &_block|
```
Comment (johntopleyons) [inline routes/home.rb]: Use string interpolation: `"https://#{CENSUS_FSDR_HOST}:#{CENSUS_FSDR_PORT}/etc"`

File: views/field_worker_hr.erb [hunk 3]
```diff
@@ -0,0 +1,401 @@
+<%= erb :menu, locals: { pageid: 'home' } %>
+
+<% unless field_worker_details.nil? %>
```
Comment (johntopleyons) [inline views/field_worker_hr.erb]: You can use `if field_worker_details.any?`

File: views/layout.erb [hunk 4]
```diff
@@ -107,7 +107,7 @@
 
             <% [:error, :notice].each do |key| %>
             <% if flash[key] %>
-              <p class="message"><%= flash[key] %></p>
+              <p class="message" ><b><%= flash[key] %></b></p>
```
Comment (johntopleyons) [inline views/layout.erb]: Use `<strong>` rather than `<b>`.

File: routes/home.rb (hunk 2 above)
Comment (PhilSharland) [reply routes/home.rb]: Yes, like I did lower down!! inconsistency!!

File: routes/home.rb (hunk 1 above)
Comment (PhilSharland) [reply routes/home.rb]: done

File: views/layout.erb (hunk 4 above)
Comment (PhilSharland) [reply views/layout.erb]: done

File: views/field_worker_hr.erb (hunk 3 above)
Comment (PhilSharland) [reply views/field_worker_hr.erb]: I think I tried that originally but the response can be nil, which means I get...

undefined method `any?' for nil:NilClass

**Pull request merged**